### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by Scriptkiddi**
41.2%

### CODE REVIEW

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/core/impl/JcrModelRepository.java [hunk 1]
```diff
@@ -101,6 +102,8 @@
 
   private static Logger logger = Logger.getLogger(JcrModelRepository.class);
 
+  private static String VORTO_NODE_TYPE = "vorto:type";
```
Comment (aedelmann) [inline repository/repository-core/src/main/java/org/eclipse/vorto/repository/core/impl/JcrModelRepository.java]: can be final, as it is not changed , correct ?

File: repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ */
+package org.eclipse.vorto.repository.web.core;
+
+import org.eclipse.vorto.repository.server.it.AbstractIntegrationTest;
+import org.eclipse.vorto.repository.server.it.TestModel;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class PayloadMappingControllerTest extends AbstractIntegrationTest {
```
Comment (aedelmann) [inline repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java]: This test class seems not be done as it contains empty test methods. Am I missing sth ?

Comment (Scriptkiddi) [reply repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java]: this is a work in progress branch, to track what we are currently each working on

File: repository/repository-core/src/test/java/org/eclipse/vorto/repository/core/impl/JcrModelRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import org.eclipse.vorto.model.ModelId;
+import org.eclipse.vorto.repository.AbstractIntegrationTest;
+import org.eclipse.vorto.repository.core.ModelNotFoundException;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.junit.Assert.*;
+
+public class JcrModelRepositoryTest extends AbstractIntegrationTest {
```
[the remark this answers is not in the document]
Comment (Scriptkiddi) [reply repository/repository-core/src/test/java/org/eclipse/vorto/repository/core/impl/JcrModelRepositoryTest.java]: this should be removed I will tomorrow

File: repository/repository-core/src/main/java/org/eclipse/vorto/repository/core/impl/JcrModelRepository.java (hunk 1 above)
Comment (Scriptkiddi) [reply repository/repository-core/src/main/java/org/eclipse/vorto/repository/core/impl/JcrModelRepository.java]: will fix tomorrow

File: repository/repository-core/src/test/java/org/eclipse/vorto/repository/core/impl/JcrModelRepositoryTest.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Scriptkiddi) [reply repository/repository-core/src/test/java/org/eclipse/vorto/repository/core/impl/JcrModelRepositoryTest.java]: Oh forgot that will remove it tomorrow

File: repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java (hunk 2 above)
Comment (Scriptkiddi) [reply repository/repository-server/src/test/java/org/eclipse/vorto/repository/web/core/PayloadMappingControllerTest.java]: this is still a in progress thing should contain test tomorrow night

**Comment by Scriptkiddi** [comment]
all done

**Pull request merged**